plugins/inputs/kube_inventory/README.md

```diff
@@ -61,19 +61,9 @@ See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
   ## Node name to filter to. No filtering by default.
   # node_name = ""
 
-  ## Use bearer token for authorization. ('bearer_token' takes priority)
-  ##
+  ## Use bearer token for authorization.
   ## Ignored if url is empty and in-cluster config is used.
-  ##
-  ## If both of these are empty, we'll use the default serviceaccount:
-  ## at: /var/run/secrets/kubernetes.io/serviceaccount/token
-  ##
-  ## To auto-refresh the token, please use a file with the bearer_token option.
-  ## If given a string, Telegraf cannot refresh the token periodically.
   # bearer_token = "/var/run/secrets/kubernetes.io/serviceaccount/token"
-  ## OR
-  ## deprecated in 1.24.0; use bearer_token with a file
-  # bearer_token_string = "abc_123"
```

Comment:
I'm not sure if this deprecation makes a lot of sense actually now that we can use secret-stores... The removal will take away that ability. What do you think?

Reply:
`bearer_token_string` is a regular string, not a secret. Also, it doesn't refresh if it would..

Reply:
But we can convert the string to a secret easily. If we now remove it, we need to add it back in later...

Reply:
I'm not removing any functionality, just removing deprecated settings from the sample configs.. 🤷

Reply:
I don't want to discuss this and will simply add the same setting back later...